### ISSUE TITLE
Header/gutter doesn't meet contrast requirements

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-hexeditor#23]
![image](https://user-images.githubusercontent.com/2193314/84065632-870af280-a979-11ea-9b36-b4e2eb6f177d.png)

**Comment by lramos15** [comment]
It used to be a bright white, but I received feedback that it was too difficult to tell the difference between the header and the editor. Do you have any suggestions for better colors?

**Comment by Tyriar** [comment]
There's the active gutter line foreground color which might be better? Also the border around the headers and how dark the headers are in general looks a bit off.

**Comment by lramos15** [comment]
I'm going to leave this open for now. I wan to investigate it further, I did add a fix to get the contrast of the header up to 6.45 it only fails the WCAG AAA for normal text which wants a 7. The gutter is the same as VS Code so I'm not sure what the best improvement is for that. I also don't really like the orange border used in high contrast mode with the orange text.

**Comment by lramos15** [comment]
After further investigation it seems our contrast ratios are similar to the editors and pass the accessibility tool
